src/_actions/asset.actions.js

```diff
@@ -33,15 +37,39 @@ export const createAsset = (assetDetail) => {
  *
  * @return {object} type and payload
  */
-export const createAssetSuccess = (asset) => {
-  return { type: CREATE_ASSET_SUCCESS, payload: asset };
-};
+export const createAssetSuccess = asset => (
+  { type: CREATE_ASSET_SUCCESS, payload: asset }
+);
 
 /**
  * createAssetFail - create new asset fail action creator
  *
  * @return {type} type and payload
  */
-export const createAssetFail = () => {
-  return { type: CREATE_ASSET_FAIL };
-};
+export const createAssetFail = () => ({ type: CREATE_ASSET_FAIL });
+
+/**
+ * load asset detail thunk
+ *
+ * @param {string} assetSerialNumber serial number of the asset
+ *
+ * @return {object} dispatch object
+ */
+export const getAssetDetail = assetSerialNumber => (
+  (dispatch) => {
```

Comment:
No need for the parenthesis. This can be `dispatch => {`

Reply:
I think for curly braces the parenthesis is required. When I try to change it, I get this ESlint error `Expected parentheses around arrow function argument having a body with curly braces. (arrow-parens)`